config/crd/v1/bases/velero.io_backups.yaml

```diff
@@ -37,6 +37,11 @@ spec:
           spec:
             description: BackupSpec defines the specification for a Velero backup.
             properties:
+              csiSnapshotCreationTimeout:
```

Comment:
Suggestion: Isn't the name `csiSnapshotCreationTimeout` a bit long ? Can we drop the word `create` ? Maybe just `csiSnapshotsTimeout` is sufficient ? WDYT ?

Reply:
I agree. Added Creation in name is due to I think it seems need to make difference from restore disk from CSI snapshot timeout setting. After discussion, looks like there is no need to add a configuration for restore CSI snapshot time, so it would be OK to use name as csiSnapshotTimeout.